libs/blog/newsletter/feature-newsletter/src/lib/newsletter-error/newsletter-error.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col items-center justify-center text-center">
+  <div class="text-al-pink mb-2 mt-5 aspect-square h-20">
```

Comment:
FYI: I decided NOT TO use our `al-icon` component because in this case we have to get this icon on init. Al icon gets svgs dynamically so when connection is broken, the icon will not be gotten (what is obvious). What is more, I also decided not to create another component for this single icon to keep to the `KISS` rule.